tests/test_date_pyparser.py

```
@@ -3,997 +3,1046 @@
-                DOYS.append((dstat, dword + '.5', dval + 0.5))
+##########################################################################################
+# Parametrized tests for ISO_DATE_PYPARSERS
+##########################################################################################
 
-        if padding:
-            before = '  '
-            after = '  '
-        else:
-            before = ''
-            after = ''
+@pytest.mark.parametrize('extended,floating,doy_flag', [
```

Ruff (0.15.6)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)